test/pool.js

```diff
@@ -425,6 +425,9 @@ test('pool dispatch', (t) => {
       },
       onComplete () {
         t.strictEqual(buf, 'asd')
+      },
+      onError (err) {
+        t.strictEqual(err.code, 'UND_ERR_INVALID_ARG')
```

Comment:
This breaks the test. It's testing something different now.

Reply:
Fixed this